CHANGELOG.md

```diff
@@ -6,6 +6,11 @@
 
 ##### Enhancements
 
+* Add `PrivateOutletRule` Opt-In rule to enforce `@IBOutlet`
+  instance variables to be `private`.  
+  [Olivier Halligon](https://github.com/AliSoftware)
+  [#754](https://github.com/realm/SwiftLint/pull/754)
```

Comment:
Please remove this line. The contributing doc states that only issues _addressed_ by this PR should be listed here: https://github.com/realm/SwiftLint/blob/master/CONTRIBUTING.md#tracking-changes